index.html

```diff
@@ -0,0 +1,263 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
+    <link rel="stylesheet" href="./main.css" type="text/css" media="all">
+    <link rel="stylesheet" href="./background-colors.css" type="text/css" media="all">
+    <title>Daniel's Pixel Art Maker</title>
+  </head>
+  <body>
+
+<main>
+  <header>
+    <img src="square-bee-icon.png"></img><h1>A Slow Bee Pixelshop</h1>
+  </header>
+  <section class="top">
+      <pixelCanvas>
+      </pixelCanvas>
+    <buttons>
+      <saveButton>Save</saveButton>
+      <loadButton>Load</loadButton>
+      <resetButton>Reset</resetButton>
+    </buttons>
+  </section>
+
+
+  <section class="bottom">
+    <palette>
+      <paletteCol>
+        <colorTile class='colorfbfcfc'></colorTile>
```

Comment:
All of this should totally be done programmatically. Almost 200 lines of the same code over and over again. 💧

If you find yourself doing that you should DRY it up. 🌵